enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/EditableDashboard.styled.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "@emotion/styled";
 import { PublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
 
 export const StyledPublicComponentWrapper = styled(PublicComponentWrapper)`
-  height: 100vh;
+  min-height: 100vh;
```

Comment:
We might want to note in the docs that the `EditableDashboard` will take up the entire viewport space, no matter what height they set it to. 

I think some customers might try to do `style={{height: "400px"}}` or try to wrap the `EditableDashboard` in a flex container with preset height, and they might be confused why that does not work and the content below `EditableDashboard` becomes obscured.

![CleanShot 2567-10-30 at 14 36 04@2x](https://github.com/user-attachments/assets/3339eb0d-60e4-4efc-852c-dcb41e012551)